public_html/wp-content/mu-plugins/blocks/assets/src/sponsors/inspector-controls.js

```diff
@@ -0,0 +1,90 @@
+/**
+ * WordPress dependencies.
+ */
+const { Component } = wp.element;
+const { InspectorControls } = wp.editor;
+const { PanelBody, PanelRow, ToggleControl, SelectControl } = wp.components;
+const { __ } = wp.i18n;
+
+/**
+ * Internal dependencies
+ */
+import GridInspectorControl from '../shared/grid-layout/inspector-control';
+import FeaturedImageInspectorControls from '../shared/featured-image/inspector-control';
+
+/**
+ * Class for defining Inspector control in sponsor block.
+ */
+class SponsorInspectorControls extends Component {
+
+	/**
+	 * Renders inspector controls.
+	 */
+	render() {
+
+		const sortOptions = [
+			{ label : __( 'Name (A to Z)', 'wordcamporg' ), value : 'name_asc' },
+			{ label : __( 'Name (Z to A)', 'wordcamporg' ), value : 'name_desc' },
+			{ label : __( 'Sponsor Level', 'wordcamporg' ), value : 'sponsor_level' },
+		];
+
+		const { attributes, setAttributes } = this.props;
+		const {
+			show_name, show_logo, show_desc, sort_by
+		} = attributes;
+
+		return (
+			<InspectorControls>
+				<GridInspectorControl
+					{ ...this.props }
+				/>
+				<PanelBody
+					title = { __( 'Content Settings', 'wordcamporg' ) }
+					initialOpen = { true }
+				>
+					<PanelRow>
+						<ToggleControl
+							label = { __( 'Name', 'wordcamporg' ) }
+							help = { __( 'Show or hide sponsor name', 'wordcamporg' ) }
+							checked = { show_name === undefined ? true : show_name }
+							onChange={ ( value ) => setAttributes( { show_name: value } ) }
+						/>
+					</PanelRow>
+					<PanelRow>
+						<ToggleControl
+							label = { __( 'Logo', 'wordcamporg' ) }
+							help = { __( 'Show or hide sponsor logo', 'wordcamporg' ) }
+							checked = { show_logo === undefined ? true : show_logo }
+							onChange={ ( value ) => setAttributes( { show_logo: value } ) }
+						/>
+					</PanelRow>
+					<PanelRow>
+						<ToggleControl
+							label = { __( 'Description', 'wordcamporg' ) }
+							help = { __( 'Show or hide sponsor description', 'wordcamporg' ) }
+							checked = { show_desc === undefined ? true : show_desc }
+							onChange={ ( value ) => setAttributes( { show_desc: value } ) }
+						/>
+					</PanelRow>
+					<PanelRow>
+						<SelectControl
+							label = { __( 'Sort by', 'wordcamporg' ) }
+							options = { sortOptions }
+							value = { sort_by || 'name_asc' }
+							onChange={ ( value ) => setAttributes( { sort_by: value } ) }
+							help = { __( 'Select whether to sort by name or sponsor level. Order of sponsor level can be configure by going to Sponsor -> Order Sponsor Levels admin menu.') }
```

Comment:
`MIGHT`

> Select whether to sort by name or sponsor level.

This part seems redundant, since it's obvious from the options in the dropdown.

>  Order of sponsor level can be configure by going to Sponsor -> Order Sponsor Levels admin menu.') }

Can we provide a direct link here, or does Gutenberg only allow text? If it's the latter, it might be good to open an issue suggesting they allow links for this type of use case.

We could probably also make this more succinct in order to make it easier to skim, and clutter the UI less. Maybe something like,

> [Configure sponsor levels](#).

...if a link is possible. It should probably open in a new tab, as much as i hate that.

Or,

> Configure sponsor levels from the Sponsor -> Order Sponsor Levels page.

...if a link isn't if possible.

Reply:
Added in https://github.com/WordPress/wordcamp.org/issues/41